hwdb.d/70-pointingstick.hwdb

```diff
@@ -17,6 +17,15 @@
 #    The vendor, product and version ID for a device node "eventX" is listed
 #    in /sys/class/input/eventX/device/id.
 #
+#  - Extended input driver device name, properties and DMI data match:
```

Comment:
You are not adding any new matches using this new format, so why add documentation for it here?

If you want to make sure all possible match formats are covered here, maybe just remove the existing docs and instead add some text pointing the user to the matches documentation in 60-keyboard.hwdb ?

This way you remove the duplication and avoid them getting out of sync in the future again. 

You could then also add the same text pointing to the matching docs to 60-evdev.hwdb which currently lacks this.

Reply:
> You are not adding any new matches using this new format, so why add documentation for it here?

Neither 60-evdev.hwdb nor 60-keyboard.hwdb has entry in extended format. I add the explanation for consistency and to provide for users how to modify the hwdb.

> If you want to make sure all possible match formats are covered here, maybe just remove the existing docs and instead add some text pointing the user to the matches documentation in 60-keyboard.hwdb ?
> This way you remove the duplication and avoid them getting out of sync in the future again.

That should also work. But currently we only have three files. Hence, I decided to copy the same explanation in these files. Then, each hwdb file becomes self-consistent.

> You could then also add the same text pointing to the matching docs to 60-evdev.hwdb which currently lacks this.

Done.